packages/remirror__theme/src/extension-tables-theme.ts

```diff
@@ -2,6 +2,15 @@ import { css } from '@linaria/core';
 
 import { getTheme } from './utils';
 
+const controllerSize = 12;
+const previewSelectionBorderColor = '#0067ce';
```

Comment:
Instead of hardcoding the colors, would it be better to define them as CSS variables in the theme (see https://github.com/remirror/remirror/blob/beta/packages/remirror__styles/theme.css#L18)? => e.g. `--preview-selection-border-color`.
This makes it easier to adjust those colors for users.